plugins/catalog-backend-module-ldap/src/ldap/read.ts

```diff
@@ -80,7 +80,10 @@ export async function defaultUserTransformer(
     entity.metadata.annotations![LDAP_UUID_ANNOTATION] = v;
   });
   mapStringAttr(entry, vendor, vendor.dnAttributeName, v => {
-    entity.metadata.annotations![LDAP_DN_ANNOTATION] = v;
+    entity.metadata.annotations![LDAP_DN_ANNOTATION] = getCaseSensitivityValue(
```

Comment:
Throughout this file, I wonder - do we _really_ want to mess up the original values as stored in the entity? I thought we wanted to preserve exactly what the server said, BUT then do the membership mapping case insensitively internally.

Reply:
The issue is that half the mapping checks pull the value from the annotation, and then other parts use the actual value, re-mapping/changing the complicated loop system seemed far more troublesome for something we're forcing.

Reply:
I didn't need to modify the annotation on the `users` entity/dn because it was only related to the `group` which is why I left it as the value provided and tested it that way.